### ISSUE TITLE
change wording on award

### ISSUE THREAD

**Issue by lcerrato** [OpenGreekAndLatin/First1KGreek#886]
I am embarrassed to be so picky, but could we remove the "A" from the Google credits?

**A Google Digital Humanities Award** s/b **Google Digital Humanities Award**

I am guess this is sourced in the text headers, so perhaps a global change there?

Thanks.

**Comment by lcerrato** [comment]
I'm guessing that the Word Counts by Source at the bottom of the OGL page http://opengreekandlatin.github.io/First1KGreek/  is auto compiled from the metadata.

Perhaps we could revisit this at some point? I think that in terms of funders, most want more detailed information in a credit line. so "Andrew W. Mellon Foundation" is an odd shorthand — if these originated via a Mellon sponsored project, there is still credit due to Tufts/Perseus and the grant itself is not mentioned. So these are really funders/sponsors not sources per se and it under-represents Tufts and Perseus.

I know this something no one should manage by hand so if the wording could be tweaked in the future, that would be great.

**Comment by sonofmun** [comment]
@lcerrato I am more than willing to change the wording to reflect what this list actually represents. If I recall, the purpose here was to make sure that funders who made the work possible are recognized. @gregorycrane What is the best way forward with this?

**Comment by lcerrato** [comment]
Thanks @sonofmun. Happy to revisit at next meeting with the group. These are certainly "sponsors" so perhaps source s/b sponsor?

**Comment by sonofmun** [comment]
@lcerrato I have finally updated the table at the bottom, deciding to actually split it into a table for the source of the text and a table for the funder. I think that is the best way to please both sides. Feel free to take a look at the Github page and give me feedback on what you think.
I have not done anything about changing "A Google ..." to just "Google...", the problem that started this whole issue. I will need to go through and actually change it in the files, which I will eventually do once Alison is done going through all the headers.

**Comment by lcerrato** [comment]
@sonofmun Thanks. Looks good to me.
